packages/foundation/src/atoms/core/shadows.css

```diff
@@ -1,4 +1,9 @@
-html * {
-    --o-ui-shadow-100: 0 4px 12px 0 hsla(223, 13%, var(--o-ui-shadow-lightness), var(--o-ui-shadow-opacity));
-    --o-ui-shadow-200: 0 8px 24px 0 hsla(223, 13%, var(--o-ui-shadow-lightness), var(--o-ui-shadow-opacity));
+.o-ui-dark-color-scheme {
+    --o-ui-global-shadow-100: 0 4px 12px 0 hsla(223, 13%, 3%, 1);
+    --o-ui-global-shadow-200: 0 8px 24px 0 hsla(223, 13%, 3%, 1);
+}
+
+.o-ui-light-color-scheme {
+    --o-ui-global-shadow-100: 0 4px 12px 0 hsla(223, 13%, 56%, .32);
+    --o-ui-global-shadow-200: 0 8px 24px 0 hsla(223, 13%, 56%, .32);
```

Comment:
I believe our boundaries are wrong here. It would really help is clarified in the RFC.

I doubt foundation should switch variable based on the current color scheme. It should only offer tokens (or const if you prefer).

Doesn't make sense to me that a global is color scheme sensitive, it seems to break the whole concept.

Reply:
It doesn make sense based on what you stated in your current RFC:

> Variables containing `-alias` in their name are directly affected by themes and are using global variables(see them as constant) who will not change based on the app theme.

But Spectrum also do it?

![image](https://user-images.githubusercontent.com/794579/111339244-fe2e8f00-864d-11eb-9be6-4b108c1fab81.png)

They seem to use "static" for tokens? for color tokens at least:

![image](https://user-images.githubusercontent.com/794579/111339359-143c4f80-864e-11eb-842d-76758b9c4549.png)

Reply:
Maybe your interpretation of "alias" vs "globals" need to be further improved? It might be alright, I am trying to make sense of all this.

Reply:
Alias are meant to be abstractions on top of our foundations, aka text-secondary, border-invalid, etc. They are often affected by a theme, but could not change. 

What is important to remember is that a global variable is static, it is used by aliases/semantic variables but they are a constant.